community/modules/scripts/spack-execute/templates/execute_commands.yml.tpl

```diff
@@ -23,7 +23,7 @@
     block:
     - name: Print commands to be executed
       ansible.builtin.debug:
-        msg: "{{ commands.split('\n') }}"
+        msg: "{{ commands.split('\n') | to_nice_yaml }}"
```

Comment:
Is there any reason to fully qualify `to_nice_yaml`?

Reply:
updated